lib/ansible/module_utils/network/common/facts/facts.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, module):
         self._connection = get_resource_connection(module)
 
         self.ansible_facts = {'ansible_network_resources': {}}
-        self.ansible_facts['ansible_gather_network_resources'] = list()
+        self.ansible_facts['ansible_net_gather_network_resources'] = list()
```

Comment:
what's the point on changing this? Will this just be part of another repo?

Reply:
It is a typo, `ansible_net_*` is added in facts key as a convention. It is similar to `ansible_net_gather_facts`